File: Java - Mundos/Mundo 1/Exercicio004.java
/*
 * EXERCÍCIO 004: Dissecando uma Variável
 * Faça um programa que leia algo pelo teclado e mostre na tela o seu tipo primitivo
 * e todas as informações possíveis sobre ele.
 */
import java.util.Scanner;
 public class Exercicio004 {
    public static void main(String[] args) {
        Scanner teclado = new Scanner(System.in);
        System.out.print("Digite algo: ");
        String n = teclado.nextLine();
        teclado.close();
        System.out.println("Está em branco: " + n.isBlank());
        System.out.println("Tamanho: " + n.length());
        System.out.println("Todo em Maiusculo: " + n.toUpperCase());
        System.out.println("Todo em Minusculo: " + n.toLowerCase());
    }
}

File: JavaScript - Mundos/Mundo 1/Exercicio004.js
/*
 * EXERCÍCIO 004: Dissecando uma Variável
 * Faça um programa que leia algo pelo teclado e mostre na tela o seu tipo primitivo
 * e todas as informações possíveis sobre ele.
 */

var frase = prompt('Digite algo: ')
document.write(`É do tipo: ${typeof(frase)} <br>`)
document.write(`É um uma frase: ${isNaN(frase)} <br>`)
document.write(`Tem ${frase.length} letras <br>`)
document.write(`Em maiusculo: ${frase.toUpperCase()} <br>`)
document.write(`Em minusculo: ${frase.toLowerCase()} <br>`)